sources/pmm.2.29.1.pmm-server.json

```diff
@@ -0,0 +1,44 @@
+{
+  "versions": [
+    {
+      "operator": "2.29.1",
+      "product": "pmm-server",
+      "matrix": {
+        "pxcOperator": {
+          "1.9.0": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.9.0",
+            "image_hash": "3783173df8b833028184de686cc98890179a249c7ed5cde8391cdfd751cf990d",
+            "status": "available",
+            "critical": false
+          },
+          "1.10.0": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0",
+            "image_hash": "73d2266258b700a691db6196f4b5c830845d34d57bdef5be5ffbd45e88407309",
+            "status": "recommended",
+            "critical": false
+          }
+        },
```

Comment:
do we need 1.11.0 operator version here?

Reply:
@beatahandzelova have we tested PXC 1.11.0 with the latest PMM?

Reply:
@nmarukovich let's merge as is and if it will be required add 1.11.0 for both 2.29.0 and 2.29.1 later

Reply:
@BupycHuk yes